### PR TITLE
Translate zh/sql/statements  index and use document

### PR DESCRIPTION
**Pull request by ghost**
Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...
Translate to zh language

Detailed description / Documentation draft:
...

> By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

> If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

> Information about CI checks: https://clickhouse.tech/docs/en/development/continuous-integration/

### CODE REVIEW

File: docs/en/sql-reference/statements/use.md [hunk 1]
```diff
@@ -3,14 +3,14 @@ toc_priority: 53
...
 
 ``` sql
 USE db
 ```
 
-Lets you set the current database for the session.
+用于设置会话的当前数据库。
 
-The current database is used for searching for tables if the database is not explicitly defined in the query with a dot before the table name.
+如果数据库未在查询中显式定义，则当前数据库用于搜索表，并在表名前加上一个点。
```
Comment (hYuang) [inline docs/en/sql-reference/statements/use.md:14]: 如果数据库.表名(database.tablename)未在查询中显式定义，则当前数据库用于搜索。
感觉 并在表名前加上一个点 这段话有点别扭

Comment (cnmade) [reply docs/en/sql-reference/statements/use.md:14]: 欢迎给出更好的意见和翻译。在不破坏原英文意思的前提下，正确的翻译。

Comment (cnmade) [reply docs/en/sql-reference/statements/use.md:14]: 如果数据库没有在查询中明确定义，并在表名前加点，则使用当前数据库来搜索表。 

这样翻译呢？对不对

Comment (cnmade) [reply docs/en/sql-reference/statements/use.md:14]: T3: 如果查询语句中 未能在表名前加点 指明数据库名， 则用当前数据库进行搜索。   表前加点指明数据库的示例: databaseA.tableX

Comment (cnmade) [reply docs/en/sql-reference/statements/use.md]: ping @hYuang 已经修了一版，看看新的翻译能否合适

**Comment by cnmade** [comment]
Hello @kssenii   would you please review this pull request?

File: docs/en/sql-reference/statements/use.md (hunk 1 above)
Comment (hYuang) [reply docs/en/sql-reference/statements/use.md]: 好多了

**Review by kssenii — APPROVED** [review]

**Comment by cnmade** [comment]
Thanks for your great job, @kssenii

**Pull request merged**